docs/nconnect-get-started.md

```diff
@@ -0,0 +1,144 @@
+---
+id: nconnect-get-started
+title: Get Started
+---
+
+nConnect allows you to securely connect to remote machines without the need for a server, public IP address, or publicly exposed ports. It features end-to-end encryption for the highest level of security and multi-path aggregation for maximum throughput.
+
+Let's connect two devices using nConnect. We'll use `ping` and `ssh` to test the connection. Any two devices that can run nConnect will do: cloud server, home desktop, embedded Linux device, NAS device, mobile phone, etc.
+
+Both devices can be in different locations, home or office, cafe shop or cloud server base.
+
+The rough outline is:
+
+- Set up nConnect Server
+- Set up nConnect Client
+- **SSH** server local IP from remote client
+
+In this case we assume we need to remotely access an `Ubuntu PC' at home from our `Windows laptop' wherever we are. The Ubuntu PC does not have a public IP, it is behind the Internet gateway at home. And our Windows laptop also has no public IP. We can access Ubuntu server from home, cafe, office or airport, anywhere we can connect to Internet.
+
+## Preparation
+
+In this simple case, we prepare a Linux(Ubuntu) server, which is located at home, and a Windows laptop, which is a mobile working device, sometimes we use it at home, sometimes we use it at office, and sometimes we use it at cafe shop.
+
+Both the Ubuntu server and the Windows laptop have no public IP, and both devices run behind some Internet access gateway. In our case, we want to enable the mobile Windows laptop to access the home Ubuntu server anytime, anywhere when Internet is available.
+
+It's very easy by installing nConnect on both devices. Let's do it.
+
+## Setting up nConnect Server on an Ubuntu Server
+
+In this case, the Ubuntu server will be remotely accessed by a Windows laptop. To remotely access our Ubuntu PC, we first need to set up nConnect Server on the Ubuntu server.
+Follow the steps below to install nConnect Server on Ubuntu:
+
+- Download the nConnect Linux binary release from the nConnect repository.
+  You can always download the latest nConnect from the nConnect github repository:
+  [nConnect release](https://github.com/nknorg/nconnect/releases)
+
+  Let's download nConnect for Ubuntu:
+
+  ```
+  $ mkdir nconnect
+  $ cd nconnect
+  $ wget https://github.com/nknorg/nconnect/releases/download/v1.1.0/linux-amd64.tar.gz
```

Comment:
I think it's better not to hard code version in doc

Reply:
Yes, modified.